### PR TITLE
add front matter for rendering a hugo website 

### PR DESCRIPTION
**Pull request by flynnduism**
This PR adds a small bit of metadata to each page, in order to structure the markdown content for rendering as a website:

The site is a [work in progress](https://github.com/deislabs/cnab.io/pull/10) - [github.com/deislabs/cnab.io/pull/10](https://github.com/deislabs/cnab.io/pull/10)

It will fetch the contents of this repo and import it into cnab.io - rather than adding Hugo _config/theme/noise_ to this repo.

### CODE REVIEW

File: 804-repositories.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+title: Registries
```
Comment (technosophos) [inline 804-repositories.md]: Is registries on a `weight` loss program?

File: 805-well-known-custom-actions.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+title: Well known custom actions
```
Comment (technosophos) [inline 805-well-known-custom-actions.md]: Also seems to have lost some `weight`

File: 804-repositories.md (hunk 1 above)
Comment (flynnduism) [reply 804-repositories.md]: the file `804-` is redundant, as the content is now in file `200-` so I've purposely left it out of the website indexing here.

File: 805-well-known-custom-actions.md (hunk 2 above)
Comment (flynnduism) [reply 805-well-known-custom-actions.md]: Thanks for catching! Updated.

**Pull request merged**